src/model/writer.js

```diff
@@ -925,24 +925,35 @@ export default class Writer {
 	 *		const position = new Position( root, path );
 	 *		writer.setSelection( position );
 	 *
-	 * 		// Sets collapsed range at the given offset in element.
+	 * 		// Sets range at the position of given node and offset.
 	 *		const paragraph = writer.createElement( 'paragraph' );
 	 *		writer.setSelection( paragraph, offset );
 	 *
+	 *		// Sets range inside the node.
+	 *		const paragraph = writer.createElement( 'paragraph', { backward } );
```

Comment:
What `backward` property of an element do? ;)

Reply:
Hmm, that's a good question! 😄

Reply:
Also, I think that we should skip this line. It is not correct to create a selection on a newly created element, which is not yet attached to the tree.